packages/documentation-site/patternfly-docs/content/demos/Compass/Compass.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Compass
+section: PatternFly-AI
+subsection: Generative UIs
+---
+
+## Demo
+
+import { useState, memo, useCallback, useRef, useEffect } from 'react';
+
+import { ActionsColumn } from "@patternfly/react-table";
+import { DataViewTable } from "@patternfly/react-data-view/dist/dynamic/DataViewTable";
+import { DataViewToolbar } from "@patternfly/react-data-view/dist/dynamic/DataViewToolbar";
+import { DataViewTextFilter } from "@patternfly/react-data-view/dist/dynamic/DataViewTextFilter";
+
+import { MessageBar } from '@patternfly/chatbot';
+
+import ThIcon from '@patternfly/react-icons/dist/esm/icons/th-icon';
+import ListIcon from '@patternfly/react-icons/dist/esm/icons/list-icon';
+import EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon";
+import PlayIcon from '@patternfly/react-icons/dist/esm/icons/play-icon';
+import OutlinedPlusSquare from '@patternfly/react-icons/dist/esm/icons/outlined-plus-square-icon';
+import OutlinedCopy from '@patternfly/react-icons/dist/esm/icons/outlined-copy-icon';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+
+import wallpaperLight from './assets/wallpaper-light.png'
+import wallpaperDark from './assets/wallpaper-dark.png'
+import imgAvatar from './assets/avatar.jpg';
+
+## Demo notes
+
+To facilitate building a compass layout, there several are useful variants and props exposed on other PatternFly components that may be useful:
+
+- `isPlain` on both [DataList](/components/data-list#plain) and [Table](/components/table#table-plain). This flag will set a transparent background for these components, allowing the glass effect in a wrapping `CompassPanel` to appear as the background.
```

Comment:
Looks like this is the table link https://pf-org--pr-4848-site.surge.sh/components/table/#plain

```suggestion
- `isPlain` on both [DataList](/components/data-list#plain) and [Table](/components/table#plain). This flag will set a transparent background for these components, allowing the glass effect in a wrapping `CompassPanel` to appear as the background.
```